cognite/neat/graph/loaders/rdf_to_dms.py

```diff
@@ -40,6 +43,117 @@ def rdf2nodes_and_edges(
     data_model = DataModel.from_rules(rules)
     pydantic_models = rules_to_pydantic_models(rules)
 
+    for class_ in rules.classes:
+        if f"{rules.space}:{class_}" in data_model.containers:
+            sparql_construct_query = build_construct_query(
+                graph_store.graph,
+                class_,
+                rules,
+                properties_optional=True,
+            )
```

Comment:
This is the speedup? The new construct query?

Reply:
I figured out that I actually already had `build_construct_query` which had option to query:
- single or list of specific instances
- all instances